packages/paste-core/components/progress-steps/package.json

```diff
@@ -25,14 +25,20 @@
     "tsc": "tsc"
   },
   "peerDependencies": {
+    "@twilio-paste/anchor": "^12.0.0",
```

Comment:
issue(blocking): remove these deps if they were added for story files

Reply:
build is failing without adding anchor

Reply:
hmmmm maybe try a `yarn clean:full` and then `yarn` and `yarn build`?

Reply:
it worked, thanks